My Series List/My Series List/Model/Repository/RepositoryController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  RepositoryController.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 19/11/2021.
+//
+
+import Foundation
+
+class RepositoryController {
+    
+    let repositoryApi = RepositoryApi()
+    let repositoryCoreData = RepositoryCoreData()
+    
+    func getDataFromRepositoryApi(completionHandler: @escaping ([Series]) -> Void) {
+        repositoryApi.getData(search: "squid") { series in
+            completionHandler(series)
+        }
+    }
+    
+    func getDataFromRepositoryCoreData(completionHandler: @escaping ([CoreDataSeries]) -> Void){
```

Comment:
Please rename this method as getSeries()

Reply:
All calls to this method should be respective renamed to this simple naming scheme